UnityProject/Assets/Scripts/Systems/Inventory/DynamicItemStorage.cs

```diff
@@ -1254,11 +1254,23 @@ public void OnPlayerRejoin(Mind mind)
 				playerRejoin.OnPlayerRejoin(mind);
 			}
 		}
+
+		//Gets all the Storage game objects that make up the dynamic item storage
+		var InventoryObjects = GetContainedInventorys();
```

Comment:
Whats the difference between GetItemSlotTree() and GetContainedInventorys()?

Reply:
GetItemSlotTree = all item slots it's made of
GetContainedInventorys = all item storage's it's made of of